dashboard/src/pages/hardwareDetails/HardwareDetails.tsx

```diff
@@ -223,6 +225,12 @@ function HardwareDetails(): JSX.Element {
           selectedIndexes={treeIndexes}
           updateTreeFilters={updateTreeFilters}
         />
+        <div className="mt-5">
+          <MemoizedCompatibleHardware
+            title={<FormattedMessage id="hardwareDetails.compatibles" />}
+            compatibles={data?.compatibles}
```

Comment:
This expression can return undefined, but you define `compatibles` props in `CompatibleHardware` component as `string[]`.

```suggestion
            compatibles={data?.compatibles ?? []}
```

Reply:
Actually, I believe the error I committed here was to add the `?` after data. data is already being tested for `undefined` or not being "prepared" above in the component so there's no need to add this `?` here, as far as I can tell.